### PR TITLE
Expose Model classes to Schema methods 

### LINKED ISSUE #397: Getting Model class object inside a Schema method
In Mongoose you can get a model class object inside a schema method by calling `this.model(modelName)`, as you can see [here](http://mongoosejs.com/docs/guide.html#methods). Is that implemented in Dynamoose? If not, I'll submit a PR.

### PR DESCRIPTION
**Pull request by dolsem**
Closes #397

### CODE REVIEW

File: lib/Attribute.js [hunk 1]
```diff
@@ -359,7 +359,7 @@ Attribute.prototype.toDynamo = function(val, noSet, model, options) {
   }
 
   var runSet = true;
-  if (isTimestamp && options.updateTimestamps === false) {
+  if (isTimestamp && options && options.updateTimestamps === false) {
```
Comment (fishcharlie) [inline lib/Attribute.js]: How is this related to this PR?

**Comment by dolsem** [comment]
My bad, I pushed a new commit to the same branch by mistake today, it got added to the pull request. It's all good now.
